validation/exclusions_test.go

```diff
@@ -37,20 +37,33 @@ var exclusions = map[string]map[uint64]bool{
 	ChainIDRPCTest: {
 		11155421: true, // sepolia-dev-0/oplabs-devnet-0   No Public RPC declared
 		11763072: true, // sepolia-dev-0/base-devnet-0     No Public RPC declared
+		12345:    true, // sepolia/opdeployer
```

Comment:
skipped because got this error: 
```
=== FAIL: github.com/ethereum-optimism/superchain-registry/validation TestValidation/op-deploy-sepolia-test_(12345)/ChainID_RPC (3.20s)
    chainid-rpc_test.go:21: 
                Error Trace:    /Users/blaine/code/superchain-registry/validation/chainid-rpc_test.go:21
                                                        /Users/blaine/code/superchain-registry/validation/validation_test.go:43
                Error:          Received unexpected error:
                                operation failed permanently after 3 attempts: Post "http://awe.some.rpc": dial tcp: lookup awe.some.rpc: no such host
                Test:           TestValidation/op-deploy-sepo
```